Source/Bridge.swift

```diff
@@ -72,15 +78,14 @@ public final class Bridge: Bridgable {
 //        let replyMessage = message.replacing(data: data)
 //        callBridgeFunction("send", arguments: [replyMessage.toJSON()])
 //    }
-    @MainActor
     @discardableResult
-    func evaluate(javaScript: String) async throws -> Any {
+    func evaluate(javaScript: String) async throws -> Any? {
         guard let webView else {
             throw BridgeError.missingWebView
         }
 
         do {
-            return try await webView.evaluateJavaScript(javaScript)
+            return try await webView.evaluateJavaScriptAsync(javaScript)
         } catch {
```

Comment:
When using async/await version of `evaluateJavaScript`, `WKWebView` expects JavaScript to return with a value (something other than Void). If there's no value returning from the JavaScript that you evaluate it will crash because it trying to unwrap a nil value to Any.